charts/redpanda/values.go

```diff
@@ -168,10 +169,11 @@ type Logging struct {
 }
 
 type Monitoring struct {
-	Enabled        *bool             `json:"enabled" jsonschema:"required"`
-	ScrapeInterval string            `json:"scrapeInterval" jsonschema:"required,pattern=.*[smh]$"`
-	Labels         map[string]string `json:"labels"`
-	TLSConfig      map[string]any    `json:"tlsConfig"`
+	Enabled        *bool                   `json:"enabled" jsonschema:"required"`
```

Comment:
This can be a `bool` instead of a pointer. It's marked as required and there's a default value in values.yaml.